### PR TITLE
158 vehicle location information

### PR DESCRIPTION
**Pull request by jlstpaul**
Addresses #158 adding location_ping_id fields to passenger_events and fare_transactions.

### CODE REVIEW

**Comment by jlstpaul** [comment]
@SorenSpicknall @e-lo can you tell me what I’m doing wrong here.

**Comment by SorenSpicknall** [comment]
> @SorenSpicknall @e-lo can you tell me what I’m doing wrong here.

The error you're seeing isn't your fault - a file used for validation was deleted in a previous PR, and the relevant test just didn't happen to run after that until you drafted and submitted these changes. I've opened #165 to address the missing file.

File: spec/fare_transactions.schema.json [hunk 1]
```diff
@@ -32,6 +32,12 @@
       }
     },
     {
+       "name": "location_ping_id",
+       "type": "string",
+       "title": "ID referencing vehicle_locations.location_ping_id",
+       "description": "Identifies a vehicle_location that is the location where the fare_transaction occurred"
```
Comment (botanize) [inline spec/fare_transactions.schema.json:38]: I don't think you want to use "vehicle_location" when you mean a generic "vehicle location". Maybe `Identifies a vehicle location associated with this fare transaction`?

File: spec/passenger_events.schema.json [hunk 2]
```diff
@@ -31,6 +31,12 @@
         "required": true
       }
     },
+    {
+      "name": "location_ping_id",
+      "type": "string",
+      "title": "ID referencing vehicle_locations.location_ping_id",
+      "description": "Identifies a vehicle_location that is the location where the passenger_event occurred"
```
Comment (botanize) [inline spec/passenger_events.schema.json:38]: See comment in fare_transactions.schema.json

**Review by botanize — CHANGES_REQUESTED** [review]
Hi John,

I suggest simplifying the language used to describe `location_ping_id` to avoid confusion between table names and observations.

File: spec/fare_transactions.schema.json (hunk 1 above)
Comment (jlstpaul) [reply spec/fare_transactions.schema.json]: Agreed.  Thanks.

File: spec/passenger_events.schema.json (hunk 2 above)
Comment (jlstpaul) [reply spec/passenger_events.schema.json]: Agreed. Thanks.

**Review by KatrinaMKaiser — CHANGES_REQUESTED** [review]
Learned from @SorenSpicknall today that you'll also need to update the sample CSVs

**Review by botanize — APPROVED** [review]

**Review by KatrinaMKaiser — APPROVED** [review]

**Pull request merged**